infra/ansible/playbooks/batcher.yaml

```diff
@@ -0,0 +1,143 @@
+        path: "/home/{{ ansible_user }}/.config/systemd/user/"
+        state: directory
+
+    - name: Add service to systemd
+      template:
+        src: services/batcher.service.j2
+        dest: "/home/{{ ansible_user }}/.config/systemd/user/batcher.service"
+        force: no
+
+    - name: Upload config file for batcher
+      template:
+        src: config-files/config-batcher.yaml.j2
+        dest: "/home/{{ ansible_user }}/config/config-batcher-holesky.yaml"
+      vars:
+        aligned_layer_deployment_config_file_path: "{{ lookup('ini', 'aligned_layer_deployment_config_file_path', file='ini/config-batcher.ini') }}"
+        eigen_layer_deployment_config_file_path: "{{ lookup('ini', 'eigen_layer_deployment_config_file_path', file='ini/config-batcher.ini') }}"
+        eth_rpc_url: "{{ lookup('ini', 'eth_rpc_url', file='ini/config-batcher.ini') }}"
+        eth_rpc_url_fallback: "{{ lookup('ini', 'eth_rpc_url_fallback', file='ini/config-batcher.ini') }}"
+        eth_ws_url: "{{ lookup('ini', 'eth_ws_url', file='ini/config-batcher.ini') }}"
+        eth_ws_url_fallback: "{{ lookup('ini', 'eth_ws_url_fallback', file='ini/config-batcher.ini') }}"
+        ecdsa_private_key_store_path: "{{ lookup('ini', 'ecdsa_private_key_store_path', file='ini/config-batcher.ini') }}"
+        ecdsa_private_key_store_password: "{{ lookup('ini', 'ecdsa_private_key_store_password', file='ini/config-batcher.ini') }}"
+        batcher_replacement_private_key: "{{ lookup('ini', 'batcher_replacement_private_key', file='ini/config-batcher.ini') }}"
+
+
+    - name: Upload env file for batcher
+      template:
+        src: config-files/env-batcher.j2
+        dest: "/home/{{ ansible_user }}/config/.env.batcher"
+      vars:
+        secret_access_key: "{{ lookup('ini', 'secret_access_key', file='ini/env-batcher.ini') }}"
+        region: "{{ lookup('ini', 'region', file='ini/env-batcher.ini') }}"
+        access_key_id: "{{ lookup('ini', 'access_key_id', file='ini/env-batcher.ini') }}"
+        bucket_name: "{{ lookup('ini', 'bucket_name', file='ini/env-batcher.ini') }}"
+        download_endpoint: "{{ lookup('ini', 'download_endpoint', file='ini/env-batcher.ini') }}"
+        log_level: "{{ lookup('ini', 'log_level', file='ini/env-batcher.ini') }}"
+
+    - name: Add reverse proxy to Caddy
+      become: true
+      template:
+        src: caddy/Caddyfile.batcher.j2
+        dest: "/etc/caddy/Caddyfile"
+      vars:
+        ansible_ssh_user: "{{ admin_user }}"
+        batcher_domain: "{{ lookup('ini', 'batcher_domain', file='ini/caddy-batcher.ini') }}"
+
+    - name: Copy keystore to server
+      ansible.builtin.copy:
+        src: '{{ keystore_path }}'
+        dest: /home/{{ ansible_user }}/.keystores/batcher
+        owner: '{{ ansible_user }}'
+        group: '{{ ansible_user }}'
+
+    - name: Start Batcher service
+      ansible.builtin.systemd_service:
+        name: batcher
+        state: started
+        enabled: true
+        scope: user
```

Comment:
we're missing `loginctl enable-linger {{ ansible_user }}`,  right? so user scoped systemd services can be kept running

Reply:
I think it is not needed since it is done in the setup.yaml